critiquebrainz/frontend/templates/artist/entity.html

```diff
@@ -66,13 +76,16 @@
         <a href="{{ url_for('artist.entity', mbid=artist.id) }}?release_type=other">{{ _('Other releases') }}</a>
       </li>
     </ul>
+    <div class="no_reviewed_entities">
```

Comment:
if the default is to not check the box, can you make this hidden by default instead of hiding it in js?